snapshots/animal_welfare/2023-08-01/global_hen_inventory.csv.dvc

```diff
@@ -1,18 +1,16 @@
 meta:
-  name: Global hen inventory
-  publication_year: 2022
-
-  publication_date: '2022-12-06'
-  source_name: Welfare Footprint Project
-  source_published_by: Welfare Footprint Project based on various sources.
-  url: https://welfarefootprint.org/research-projects/laying-hens/
-  source_data_url:
-  file_extension: csv
-  license_url: https://welfarefootprint.org/research-projects/laying-hens/
-  license_name: Creative Commons BY
-  date_accessed: 2023-08-01
-  is_public: true
-  description: |
+  origin:
+    dataset_title_owid: Global hen inventory
+    dataset_title_producer: Global hen inventory
```

Comment:
I guess we'll need to change these names (and others) after the renamings? Might have to do that as well for other snapshots :D

Reply:
Yes, these metadata files are in limbo...